common/button/button-en.html

```diff
@@ -0,0 +1,415 @@
+<div class="btn-group btn-group-xs">
+	<button type="button" class="btn btn-default">Left</button>
+	<button type="button" class="btn btn-default">Middle</button>
+	<button type="button" class="btn btn-default">Right</button>
+</div>
+
+<h5>Justified (links):</h5>
+<div class="btn-group btn-group-justified">
+	<a class="btn btn-default" role="button">Left</a>
+	<a class="btn btn-default" role="button">Middle</a>
+	<a class="btn btn-default" role="button">Right</a>
+</div>
+
+<h5>Justified (buttons):</h5>
+<div class="btn-group btn-group-justified">
+	<div class="btn-group">
+		<button type="button" class="btn btn-default">Left</button>
+	</div>
+	<div class="btn-group">
+		<button type="button" class="btn btn-default">Middle</button>
+	</div>
+	<div class="btn-group">
+		<button type="button" class="btn btn-default">Right</button>
+	</div>
+</div>
+
+<h4>Code sample</h4>
+<pre><code><span class="com">// Default:</span>
+&lt;div class="<strong>btn-group</strong>"&gt;
+	&lt;button type="button" class="btn btn-default"&gt;...&lt;/button&gt;
+&lt;/div&gt;
+
+<span class="com">// Large button group:</span>
+&lt;div class="btn-group <strong>btn-group-lg</strong>"&gt;...
+&lt;/div&gt;
+
+<span class="com">// Small button group:</span>
+&lt;div class="btn-group <strong>btn-group-sm</strong>"&gt;...
+&lt;/div&gt;
+
+<span class="com">// Extra small button group:</span>
+&lt;div class="btn-group <strong>btn-group-xs</strong>"&gt;...
+&lt;/div&gt;
+
+<span class="com">// Justified links:</span>
+&lt;div class="btn-group <strong>btn-group-justified</strong>"&gt;
+&lt;a class="btn btn-default" href="#" role="button"&gt;...
+&lt;/a&gt;
+&lt;/div&gt;
+
+<span class="com">// Justified button group:</span>
+&lt;div class="btn-group <strong>btn-group-justified</strong>"&gt;
+&lt;div class="<strong>btn-group</strong>"&gt;
+	&lt;button type="button" class=" <strong>btn btn-default</strong>"&gt;...
+	&lt;/button&gt;
+&lt;/div&gt;...
+&lt;/div&gt;
+</code></pre>
+
+<h3 id="bgvertical">Button groups (Vertical)</h3>
+<p>Make a set of buttons appear vertically stacked rather than horizontally with the <code>btn-group-vertical</code> class.</p>
+<h4>Working Example</h4>
+<div class="btn-group-vertical">
+	<button type="button" class="btn btn-default">Top</button>
+	<button type="button" class="btn btn-default">Middle</button>
+	<button type="button" class="btn btn-default">Bottom</button>
+</div>
+
+<h4>Code sample</h4>
+<pre><code><span class="com">// Vertical button group:</span>
+&lt;div class="<strong>btn-group-vertical</strong>"&gt;
+	&lt;button type="button" class="btn btn-default"&gt;
+	Top
+	&lt;/button&gt;
+	&lt;button type="button" class="btn btn-default"&gt;
+	Middle
+	&lt;/button&gt;
+	&lt;button type="button" class="btn btn-default"&gt;
+	Bottom
+	&lt;/button&gt;
+&lt;/div&gt;</code></pre>
+
+<h3 id="buttontoolbar">Button toolbar</h3>
+<p>Use to display multiple groups of buttons together on a single line with the <code>btn-toolbar</code> class.</p>
```

Comment:
```suggestion
<p>Use to display multiple groups of buttons together on a single line with the <code>btn-toolbar</code> class. Note the use of this feature require some additional requirement to be comform to WCAG.</p>
```

Reply:
Addressed in commit  "Button: Addressed Review Comments"